.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Feature request
+about: Suggest features or enhancements
+title: ''
+labels: enhancements
+assignees: ''
+
+---
+
+<!-- The Dependabot team is currently at reduced capacity, because of this our
+response times on issues will be slower than we'd like. -->
+
+**Describe the feature**
+A clear and concise description of the feature you would like Dependabot to
+support.
```

Comment:
Maybe just a comment like we do on feedback? Personally find it a bit annoying to remove text from the issue when adding a new one :) 

```suggestion
<!-- Describe the feature you'd like. -->
```